stdlib/2and3/curses/textpad.pyi

```diff
@@ -1,11 +1,11 @@
 from _curses import _CursesWindow
-from typing import Callable, Union
+from typing import Callable, Optional, Union
 
 def rectangle(win: _CursesWindow, uly: int, ulx: int, lry: int, lrx: int) -> None: ...
 
 class Textbox:
     stripspaces: bool
-    def __init__(self, w: _CursesWindow, insert_mode: bool = ...) -> None: ...
-    def edit(self, validate: Callable[[int], int]) -> str: ...
+    def __init__(self, win: _CursesWindow, insert_mode: bool = ...) -> None: ...
```

Comment:
This one isn't either `    def __init__(self, win, insert_mode=False):`.

Reply:
It isn't positional-only in the stub either :-)

Reply:
um, I guess I just saw one underscore and assumed there were underscores everywhere